src/lib.rs

```diff
@@ -284,6 +284,12 @@ pub struct OfferInfo {
     pub status: OfferStatus,
 }
 
+pub struct SweepInfo {
+    pub address: String,
+    pub onchain_fee_rate: u32,
+    pub onchain_fee_sat: u64,
```

Comment:
Thinking about the use case here, I think it would make more sense to return the `onchain_fee` as an Amount.

Reply:
Yes, completely missed that, thanks!

Reply:
Fixed in [a92a7aa](https://github.com/getlipa/lipa-lightning-lib/pull/765/commits/a92a7aa585676e17f96fe46ff63b31fe07fa098b)